financial-connections/src/main/java/com/stripe/android/financialconnections/di/FinancialConnectionsSheetConfigurationModule.kt

```diff
@@ -9,39 +9,38 @@ import com.stripe.android.financialconnections.FinancialConnectionsSheet
 import dagger.Module
 import dagger.Provides
 import javax.inject.Named
-import javax.inject.Singleton
 
 @Module
 internal object FinancialConnectionsSheetConfigurationModule {
 
     @Provides
     @Named(PUBLISHABLE_KEY)
-    @Singleton
+    @ActivityRetainedScope
```

Comment:
Note that, given the new component structure, there's no room for mistakenly scoping a dependency as @Singleton on a non shared component. Dagger would fail at compile time if one of these (or other module) dependencies are scoped as @Singleton.